mlir/lib/Dialect/LLVMIR/Transforms/UseDefaultVisibilityPass.cpp

```diff
@@ -0,0 +1,49 @@
+//===- UseDefaultVisibilityPass.cpp - Update default visibility ---------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
Same here the 1 line and line 7 do not have the same length.

Reply:
Nice catch!